### ISSUE TITLE
Distraction-free mode still show toolbars

### ISSUE THREAD

**Issue by kromsam** [Zettlr/Zettlr#5472]
### Description

This issue was somewhat solved earlier (https://github.com/Zettlr/Zettlr/issues/3999) but it has returned. The top two toolbars should not be shown by default in a distraction-free mode.

![Screenshot From 2024-11-09 13-51-30](https://github.com/user-attachments/assets/35ebcdfb-d22a-46f6-930e-b4a9c688c73b)

### Reproducing

Press F11 (Full Screen)
Press ctrl+J (distraction free)

### Zettlr version

3.2.2

### Installation Method

AUR (Arch Repository)

### Your Platform

- [ ] Windows
- [ ] macOS
- [X] Linux

### Architecture

- [X] Intel 64bit
- [ ] ARM 64bit (includes Apple Silicon)

### Operating System Version

ArchLinux

**Comment by BabakFarshchian** [comment]
Still shows toolbars in beta 2. Windows 11.

**Comment by nathanlesage** [comment]
In full screen?

**Comment by BabakFarshchian** [comment]
Yes also in full screen. I can see the title bar, the menu bar, and the status bar.

**Comment by nathanlesage** [comment]
I've now done a very complicated truth table to exactly figure out when to show and not show the three relevant window chrome elements (title, menu, and tool bar) across the three operating systems and various settings and modes. Refer to the commit to see how it works by now. In most cases, there will always be one bar shown, which is non-negotiable (on Windows, for example, this is the menu bar).
